code/lib/cli/src/sandbox-templates.ts

```diff
@@ -193,7 +193,7 @@ const baseTemplates = {
     skipTasks: ['e2e-tests-dev'],
   },
   'vue3-vite/default-ts': {
-    name: 'Vue3 Vite (TS)',
+    name: 'Vue v3 Vite (TypeScript)',
```

Comment:
I wonder if this makes any difference in the `storybook sandbox` command. Can you try and check whether this still works as intended?

```
storybook sandbox vue3
```

Reply:
good catch, this still works. It uses the keys, not the names.